internal/core/queries/get_in_all.go

```diff
@@ -54,7 +56,13 @@ func (ch GetAllIntegrationQueryHandler) Handle(ctx context.Context, _ mediator.M
 			AutoPiDefaultTemplateID: im.AutoPiDefaultTemplateID,
 			RefreshLimitSecs:        v.RefreshLimitSecs,
 			TokenID:                 v.TokenID.Int,
+			Points:                  v.Points,
+		}
+
+		if !v.ManufacturerTokenID.IsZero() {
+			result[i].ManufacturerTokenID = v.ManufacturerTokenID.Big.Int(new(big.Int))
```

Comment:
You can just do 
```suggestion
			result[i].ManufacturerTokenID = v.ManufacturerTokenID.Big.Int(nil)
```
I'm pretty sure.